erica/application/FreischaltCode/FreischaltCode.py

```diff
@@ -1,19 +1,18 @@
-from datetime import date
+from pydantic import BaseModel
+from erica.domain.FreischaltCode.FreischaltCode import FreischaltCodeRequestPayload, FreischaltCodeRevocatePayload, \
+    FreischaltCodeActivatePayload
 
-from erica.application.base_dto import BaseDto
 
+class FreischaltCodeRequestDto(BaseModel):
```

Comment:
Those should still use the BaseDto I think

Reply:
Oh shit true, I will update all dto to inherit from BaseDto